lib/bolt/boltdir.rb

```diff
@@ -6,10 +6,10 @@ module Bolt
   class Boltdir
     BOLTDIR_NAME = 'Boltdir'
 
-    attr_reader :path, :config_file, :inventory_file, :modulepath, :hiera_config, :puppetfile, :rerunfile
+    attr_reader :path, :config_file, :inventory_file, :modulepath, :hiera_config, :puppetfile, :rerunfile, :type
 
     def self.default_boltdir
-      Boltdir.new(File.join('~', '.puppetlabs', 'bolt'))
+      Boltdir.new(File.join('~', '.puppetlabs', 'bolt'), 'default')
```

Comment:
Lets call this "homedir" or "user" so we can change it.

Reply:
Updated and set type to `user`.